scripts/build-dash.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+CURRENT_BRANCH=$(git rev-parse --abbrev-ref HEAD)
+
+# Checkout and sync the branch if the directory exists, otherwise clone the repository
+if [ -d "test/e2e/globalping-dash" ]; then
+  cd test/e2e/globalping-dash
+	git fetch
+  git checkout $CURRENT_BRANCH || git checkout master
+  git reset --hard @{u}
```

Comment:
```suggestion
  git add .
  git reset --hard @{u}
  git fetch
  git checkout $CURRENT_BRANCH || git checkout master
```

Reset should likely be first, otherwise the checkout may fail. `git add .` ensures all conflicting files are affected by reset.

Reply:
Added, but we still need to reset after the checkout, to effectively pick up force pushed commit overrides.